lightkurve/lightcurvefile.py

```diff
@@ -97,10 +97,10 @@ def __init__(self, path, quality_bitmask='default', **kwargs):
 
     @staticmethod
     def from_archive(target, cadence='long', quarter=None, month=None,
-                     campaign=None, **kwargs):
-        """Fetch a Light Curve File from the Kepler/K2 data archive at MAST.
+                     campaign=None, radius=1., targetlimit=1, verbose=True, **kwargs):
+        """Fetch a Lightcurve File from the Kepler/K2 data archive at MAST.
 
-        Raises an `ArchiveError` if a unique file cannot be found.  For example,
+        Raises an `ArchiveError` if a unique LCF cannot be found.  For example,
```

Comment:
We'll need a bit more text here to explain when multiple files are and aren't returned.